main.js

```diff
@@ -0,0 +1,101 @@
+let month = 0;
+
+const prevMonthButton = document.getElementById("prevMonth");
+const nextMonthButton = document.getElementById("nextMonth");
+const currentMonthYear = document.getElementById("currentMonthYear");
+const calendarDays = document.getElementById("calendarDays");
+const themeSwitch = document.getElementById('checkbox');
+const body = document.body;
+let isDarkTheme = false;
+
+// Functions
+function updateCalendar() {
+    let date = new Date();
+
+    date.setMonth(date.getMonth() + month);
+
+    // Set the displayed month and year
+    currentMonthYear.textContent = `${date.toLocaleString("en-US", { month: "long" })} ${date.getFullYear()}`;
+
+    calendarDays.innerHTML = '';
+    // Calculate the first day of the month
+    let firstDay = new Date(date.getFullYear(), date.getMonth(), 1);
+    let lastDay = new Date(date.getFullYear(), date.getMonth() + 1, 0);
+
+    // Create the previous month's days
+    let prevMonthLastDay = new Date(date.getFullYear(), date.getMonth(), 0);
+    let prevMonthDays = prevMonthLastDay.getDate();
```

Comment:
@raaj-oo7 Rename it to `lastDayOfPreviousMonth` and this variables could be constants